internal/providers/terraform/google/compute_forwarding_rule.go

```diff
@@ -0,0 +1,84 @@
+package google
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetComputeForwardingRuleRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "google_compute_forwarding_rule",
+		RFunc: NewComputeForwarding,
+		Notes: []string{"Price for additional forwarding rule is used"},
+	}
+}
+
+func GetComputeGlobalForwardingRuleRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "google_compute_global_forwarding_rule",
+		RFunc: NewComputeForwarding,
+		Notes: []string{"Price for additional forwarding rule is used"},
+	}
+}
```

Comment:
I think It would be better if you separate it (and _proxy resources too) to different files, cos it is more consistent with our codebase and 1 resource = 1 file paradigm

Reply:
I read [a comment](https://news.ycombinator.com/item?id=27468837) on HN today that made me think more about this: "To be a good X you must follow the rules. To be a great X you must have followed the rules so well as to learn why they’re there and when they should be broken." 

@ekifel, FYI @NeverovDS pinged me and asked if it seems pointless to create a lot of extra files so in this case maybe we relax our consistency rule :)